src/main/java/org/alliancegenome/curation_api/model/entities/bulkloads/BulkManualLoad.java

```diff
@@ -30,5 +30,5 @@ public class BulkManualLoad extends BulkLoad {
 
 	@JsonView({ View.FieldsOnly.class })
 	@Enumerated(EnumType.STRING)
-	private BackendBulkDataType dataType;
+	private BackendBulkDataProvider dataProvider;
```

Comment:
@oblodgett Do you mean the dataProvider object? - I don't think that would work as we don't reuse dataProvider objects, so there isn't a single object for RGD, for example

Reply:
Ok what about an Organization then?

Reply:
I'm inclined to leave it as an enum for now as I'm not sure how that would play with the UI - we don't want all organizations to be in any dropdowns.  Talking of which, aren't these changes going to break the UI?

Reply:
I think we need a change in DataLoadsComponent.js too.  Also, does the change from "HUMAN" to "OMIM" require a database migration? It will need a change to the enum as that's currently HUMAN.

Reply:
yes you are right... just added it.

Reply:
cool, I think that covers everything - I'm happy to merge if you are